buildout.cfg

```diff
@@ -61,7 +61,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.34
+snovault = git https://github.com/ENCODE-DCC/snovault.git branch=SNO-113-make-audit-view
```

Comment:
this should have been a distinct commit.  Since I am about to do the RC I can patch over it.